src/UI/templates/default/Dropdown/dropdown.less

```diff
@@ -0,0 +1,11 @@
+.dropdown .btn {
+	text-align: left;
+}
+
+.dropdown hr {
+	margin: 9px 0;
```

Comment:
Either use padding-large-vertical (defaults to 6px) here or introduce a new variable.